packages/components/src/input-control/utils.js

```diff
@@ -41,10 +41,10 @@ export function useDragCursor( isDragging, dragDirection ) {
 	useEffect( () => {
 		if ( isDragging ) {
 			document.documentElement.style.cursor = dragCursor;
-			document.documentElement.style.pointerEvents = 'none';
+			document.documentElement.style.userSelect = 'none';
```

Comment:
Not sure how important is this property or if it's just a progressive enhancement but for IE11 and safari, I might need a prefix.

Reply:
Just checked. It doesn't look like it does anything for this particular interaction.

I thought that by dragging outside the element, we would experience text selection issues (or undesired cursor interaction issues). It doesn't look like that's the case.

I'll remove it :).